examples/summarize_rlhf/sft/train_gptj_summarize.py

```diff
@@ -0,0 +1,114 @@
+import random
+
+import numpy as np
+import torch
+from summarize_dataset import TLDRDataset
+from transformers import (
+    AutoModelForCausalLM,
+    AutoTokenizer,
+    Trainer,
+    TrainingArguments,
+    default_data_collator,
+)
+
+try:
+    import evaluate
+except ImportError:
+    raise ImportError(
+        "To run this example, please install the following packages: [`evaluate`, `nltk`, `rouge_score`]."
+    )
+
+
+def set_seed(seed_val=42):
+    random.seed(seed_val)
+    np.random.seed(seed_val)
+    torch.manual_seed(seed_val)
+    torch.cuda.manual_seed_all(seed_val)
+
+
+if __name__ == "__main__":
+    output_dir = "gptj-supervised-summarize-checkpoint"
```

Comment:
Config file pls

Reply:
Note that this is the `SFT` training script that is `transformers.Trainer`-based. I don't believe `transformers` supplies a config-to-[`transformers.TrainingArguments`](https://huggingface.co/docs/transformers/main_classes/trainer#transformers.TrainingArguments) converter, so I'm okay leaving this for now.